pytest_operator/plugin.py

```diff
@@ -414,6 +423,20 @@ class ModelInUseError(Exception):
 Timeout = TypeVar("Timeout", float, int)
 
 
+def _connect_kwds(request) -> Dict[str, Any]:
+    """Create a dict of keyword arguments for connecting to a model."""
+    kwds = {}
+    if val := request.config.option.juju_max_frame_size:
+        if 0 < val <= connection.Connection.MAX_FRAME_SIZE:
```

Comment:
Let's not enforce the max.

After all, the intention of the change is to allow larger sizes than the default.

```suggestion
        if 0 < val:
```